frontend/src/pages/EnterReceiptPage/QuantityInput.tsx

```diff
@@ -11,8 +11,9 @@ const QuantityInput: React.FC<QuantityInputProps> = ({
   onQuantitiyChange
 }) => {
   const onValueChange: React.ChangeEventHandler<HTMLInputElement> = e => {
-    const asNum = parseInt(e.currentTarget.value);
-    if (!isNaN(asNum)) {
+    const asNum =
+      e.currentTarget.value === "" ? 0 : parseInt(e.currentTarget.value);
```

Comment:
Can we change e to element or something similar for easier readability?

Reply:
We could change it to event. @jhackshaw-dds what naming convention have you been using for input events?

Reply:
I think `e` is fine for now as I don't think it's consistent across the rest of the code base